tests/iDibo_tests.py

```diff
@@ -1,11 +1,24 @@
 from nose.tools import *
-import iDibo
 
-def setup():
-    print "SETUP!"
+import unittest
+from gluon.globals import Request
 
-def teardown():
-    print "TEAR DOWN!"
+db = test_db
 
-def test_basic():
-    print "I RAN!"
\ No newline at end of file
+execfile("applications/iDibo/controllers/default.py", globals())
+db(db.auth_user.id > 0).delete()
+db.commit()
+
+
+class TestRegister(unittest.TestCase):
+    def setup(self):
+        request = Request()
+
+    def test_register_title(self):
+        self.controller.register()
+        my_title = self.controller.register.response.title
```

Comment:
Hint for above: you cannot access the local variables of a function, so either there is a global way of accessing the response of the web2py request or you have to perform assertions on what you returned, which is the form. You could check if it contains "Voters registration number" and the other fields mentioned